AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -222,18 +214,56 @@ private extension SessionCartView {
                         .frame(height: 120)
                         .disabled(true)
                     HStack() {
-                            Text(startTime)
+                            startTime
                             Spacer()
                         Text("\(Strings.SessionCartView.avgSession) \(selectedStream.unitSymbol ?? "")")
                             Spacer()
-                            Text(endTime)
+                            endTime
                     }.foregroundColor(.aircastingGray)
                     .font(Font.muli(size: 13, weight: .semibold))
                 }
             }
         }
     }
     
+    var startTime: some View {
+        let formatter = Constants.dataFormatter
+        
+        if !(session.isMobile && session.isActive && session.deviceType == .MIC) {
```

Comment:
I think we have s`ession.isMic` or sth like that ;D

Reply:
done!